src/universal/components/ReflectionEditorWrapper.js

```diff
@@ -176,7 +176,12 @@ class ReflectionEditorWrapper extends Component<Props> {
 
   render() {
     const {ariaLabel, editorState, anonEditing, isBlurred, onBlur, onFocus, placeholder, readOnly} = this.props;
-    const userSelect = readOnly ? 'none' : 'text';
+    // Folks may want to copy text from reflection cards to quote in task cards, so going to allow unless blurred (TA)
+    const userSelect = (isBlurred || anonEditing) ? 'none' : 'text';
```

Comment:
we need to fix or remove the anonEditing variable. it's defined as !isBlurred, so it doesn't provide any real value.

Reply:
Yeah, the only catch that I’m not sure to work around is for a styling detail. I want normal editing styles, blurred styles, and hint color style for “Somebody is typing…” — basically I need to know if the styles are for the anon card or not and if that card is blurred or is “Somebody is typing…” hint. I don’t want any user select for the anon card. I will try to simplify it a little.

Reply:
gotcha, in that case in the AnonymouseReflectionCard i'd pass down an `isAnonymous` prop. then, you can just say userSelect = isAnonymous ? 'none' : 'text';